platform/ios/test/MGLMapViewConversionTests.m

```diff
@@ -0,0 +1,52 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+#import "../../ios/src/MGLMapView.h"
+
+@interface MGLMapViewConversionTests : XCTestCase
+
+@property (nonatomic) UIView *superView;
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation MGLMapViewConversionTests
+
+- (void)setUp {
```

Comment:
This method is called once before each `-test…` method, while `+setUp` is called once before the whole test suite. It’s probably only necessary to set up the map view once before the whole test suite. (Same with `-tearDown` below.)